staging/src/k8s.io/client-go/util/workqueue/parallelizer.go

```diff
@@ -50,3 +51,37 @@ func Parallelize(workers, pieces int, doWorkPiece DoWorkPieceFunc) {
 	}
 	wg.Wait()
 }
+
+// ParallelizeUntil is a framework that allows for parallelizing N
+// independent pieces of work until the context done or canceled.
```

Comment:
Suggest rewording "until the context done or canceled" to "until done or the context is canceled"